csunplugged/topics/content/en/binary-numbers/programming-challenges/binary-cards-representing-any-number/python-solution.md

```diff
@@ -0,0 +1,16 @@
+```python
+original = input('Please enter a decimal number:\n')
```

Comment:
Instead of a `\n` would just a space be more appropriate (it's what we teach in NZ schools and uni).